pkg/kubernetes/client/v1alpha1/client.go

```diff
@@ -0,0 +1,326 @@
+/*
```

Comment:
Is this file Required in the pkg directory or inside the vendor we can have

Reply:
I just followed the standard that `maya` directory was following. And no, vendor contains all the library that is just imported and used, but this is a package which is custom made just for our convenience. SO I think ti would better to have it in pkg